www/Themes/Centreon-2/style.css

```diff
@@ -1772,6 +1776,7 @@ span.state_badge {
     width: 18px !important;
     right: -10px !important;
     border-radius: 10px;
```

Comment:
It is not necessary to leave this line with the new line you created (border at 0)